audit_log/middleware.py

```
@@ -129,3 +134,193 @@ def process_request(self, request):
+                                   weak=False)
+            signals.post_save.connect(update_post_save_info,
+                                    dispatch_uid=(self.__class__, id(request),),
+                                    weak=False)
+        
+        async def _cleanup_signals(self, request):
+            if settings.DISABLE_AUDIT_LOG:
+                return
+            signals.pre_save.disconnect(dispatch_uid=(self.__class__, id(request),))
+            signals.post_save.disconnect(dispatch_uid=(self.__class__, id(request),))
+        
+        def _update_pre_save_info(self, user, session, sender, instance, **kwargs):
```

Ruff (0.13.1)

Unused method argument: `kwargs`

(ARG002)